### PR TITLE
Refactor the way limits are handled

### PR DESCRIPTION
**Pull request by jzuhone**
This PR refactors the way limits are handled in dashboard plots. It arises from the fact that we now keep the limits inside JSON files, and that xija now has default colors set for limit lines. 

It consolidates code into a smaller number of lines as well.

### CODE REVIEW

**Comment by jzuhone** [comment]
ping @matthewdahmer

**Comment by matthewdahmer** [comment]
@jzuhone unfortunately this broke some of my existing code, however I have a version based on this that keeps your improvements with some additions that ensure most of the plots plot correctly. The only plot that does not entirely look as good as I'd like is the ACIS FP plot when all the limits are shown, however I assume your version has this issue as well. We can work this issue further if you would like. I can push my version to your branch, or I can email it to you so you can review it first.

**Comment by jzuhone** [comment]
Hi @matthewdahmer go ahead and push yours on mine and we can sort out the ACIS FP plot to make it look better.

**Comment by matthewdahmer** [comment]
I adjusted the limit line font size to 12pts to reduce how much the labels overlap, let me know if that makes the associated text too small.

**Comment by jzuhone** [comment]
@matthewdahmer 

> The only plot that does not entirely look as good as I'd like is the ACIS FP plot when all the limits are shown, however I assume your version has this issue as well. 

I assume you mean when the -84 C limit is plotted? In general I would want to avoid plotting this limit (and the -80 C limit) unless we got close. Or we may have to adopt making one dashboard plot for observations and another for radzones.

**Comment by matthewdahmer** [comment]
@jzuhone no I meant how the labels for the different science limits are on top of each other:
ACIS_FP_FPTEMP_Model_Dashboard
I rather like having the -84C and -80C limits on there, but that's easy to remove since it's an input into the plotting function.

**Comment by jzuhone** [comment]
@matthewdahmer did you have any other recommendations?

**Comment by matthewdahmer** [comment]
@jzuhone We could just plot the -112, -111, and -109 lines without labels, or use a generic label like "ACIS FP Science Limits (-112C, -111C, -109C)" shown above the -109C limit line. There probably should be some logic to assemble the limit text to reflect the actual lines shown (in case the -112 limit is missing for example).

**Comment by matthewdahmer** [comment]
@jzuhone I actually have an implementation of the second option, should be ready later tonight.

**Comment by matthewdahmer** [comment]
How does this look to you? If you like it I'll push the code.

ACIS_FP_FPTEMP_Model_Dashboard

**Comment by jzuhone** [comment]
Looks great!

File: xijafit/dashboard.py [hunk 1]
```diff
@@ -179,114 +183,87 @@ def dashboard(prediction, tlm, times, limits, modelname='PSMC', msid='1pdeaat',
...
+        # if a limit is extreme compared to the available data or no value is entered, skip over it
+        if value is None or value < ylim1[0] - 10 or value > ylim1[1] + 10:
+            continue
+
+        ax1.axhline(value, color=get_limit_color(name), linewidth=1.5)
+
+        limit = get_limit_spec(name)
+
+        qualifier = str(limit["qualifier"])
+        if (qualifier == "acisi") or (qualifier == "aciss") or (qualifier == "aciss_hot"):
```
Comment (jzuhone) [inline xijafit/dashboard.py]:
```suggestion
        if qualifier in ["acisi", "aciss", "aciss_hot"]:
```

Comment (jzuhone) [reply xijafit/dashboard.py]: I think the above suggestion should also work.

**Comment by jzuhone** [comment]
This looks good!

**Pull request merged**